src/bin/find.js

```diff
@@ -8,6 +8,7 @@ var options = {
   getAllAvailableRules: ['all-available', 'a'],
   getUnusedRules: ['unused', 'u'],
   n: ['no-error'],
+  noCore: ['nocore', 'C'],
```

Comment:
I would suggest, keep the PR simple, don't worry about changing the `options`. It could be a separate PR all together. 

Let this PR concentrate on not listing the core rules and revisit the options in later commits.